www/main/viewer/utils/chromecast.js

```diff
@@ -1,182 +1,192 @@
-const remote = require("@electron/remote");
+/* global chrome */
+import { ipcRenderer } from 'electron';
+
+const remote = require('@electron/remote');
+
+/* eslint-disable global-require */
+export const tingle = require('../../../assets/js/vendor/tingle');
```

Comment:
Linter didnt allow exporting a module imported with require. But this is being used at multiple places so I added an exception.

Reply:
Okay veerji. But you can just disable this for just next line.